Jenkinsfile_CNP

```diff
@@ -29,7 +29,9 @@ def secrets = [
   'unspec-${env}': [
     secret('microservicekey-unspec-service', 'S2S_SECRET'),
     secret('ccd-importer-username', 'CCD_CONFIGURER_IMPORTER_USERNAME'),
-    secret('ccd-importer-password', 'CCD_CONFIGURER_IMPORTER_PASSWORD')
+    secret('ccd-importer-password', 'CCD_CONFIGURER_IMPORTER_PASSWORD'),
+    secret('postgres-user', 'POSTGRES_USER'),
+    secret('postgres-password', 'POSTGRES_PASSWORD')
```

Comment:
Hey Sam, How we are adding these vault values in unspec vaults, manually or copying from another camunda vault?

Reply:
manually